packages/components/CHANGELOG.md

```diff
@@ -26,6 +26,10 @@
 
 -   `SlotFill`: rewrite the non-portal version to use `observableMap` ([#67400](https://github.com/WordPress/gutenberg/pull/67400)).
 
+### Documentation
+
+-   `Badge`: Fix Storybook url link ([#68077](https://github.com/WordPress/gutenberg/pull/68077)).
```

Comment:
Doesn't need a CHANGELOG entry IMO, but up to you.

Reply:
I don't have a strong opinion either, I'll delete this and then merge 👍